test/e2e/diagram.navigation.fit.test.ts

```diff
@@ -18,7 +18,7 @@ import type { MatchImageSnapshotOptions } from 'jest-image-snapshot';
 import 'jest-playwright-preset';
 import { join } from 'node:path';
 import type { Page } from 'playwright';
-import { FitType } from '../../src/component/options';
+import { FitType } from '@lib//component/options';
```

Comment:
**question:** Is it normal it is `@lib//` and not `@lib/`? (same for the other files)

Reply:
No this is an error (I guess my search/replace action adds the extra slash). I have detected it in #2662 where the problem is fixed.
I will fixed it here.

Reply:
@csouchet done in ef0a2ac3